quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/DashboardController.java

```diff
@@ -48,4 +62,19 @@ public DashboardController(DashboardRepository repository, DashboardMapper dashb
         return new ResponseEntity<>(resources, OK);
     }
 
+    private DashboardConfig addAllWidgetsToDashboard(DashboardConfig dashboard) {
+        // Workaround until Dashboard is assosiated with widgets
+        dashboard.setWidgetConfigs(widgetConfigRepository.findAll());
+        return dashboard;
+    }
+
+
+    @RequestMapping(method = POST)
+    public ResponseEntity<HalJsonResource> create(@RequestBody DashboardConfig dashboardConfig) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)